tests/integration/pox/pox-3/direct-and-pooled.spec.ts

```diff
@@ -0,0 +1,283 @@
+    expect(poxInfo.current_cycle.stacked_ustx).toBe(900_000_000_000_000);
+    // FIXME: Is this right? Alice has stacked 75M and she gets unlocked
+    expect(poxInfo.current_cycle.min_threshold_ustx).toBe(75_000_000_000_000);
+
+    // All STX are unlocked in the next cycle
+    expect(poxInfo.next_cycle.stacked_ustx).toBe(0);
+
+    // Check Alice's table entry for cycle 2 -- should be not found
+    let poxAddrInfo0 = await readRewardCyclePoxAddressForAddress(
+      network,
+      3,
+      2,
+      Accounts.WALLET_1.stxAddress
+    );
+    expect(poxAddrInfo0).toBeNull();
+
+    // Check Faucets's table entry for cycle 2
+    let poxAddrInfo1 = await readRewardCyclePoxAddressForAddress(
+      network,
+      3,
+      2,
+      Accounts.FAUCET.stxAddress
+    );
+    expect(poxAddrInfo1?.["total-ustx"]).toEqual(uintCV(900_000_000_000_000));
+
+    // Check Alice's table entry for cycle 3 -- should be not found
+    poxAddrInfo0 = await readRewardCyclePoxAddressForAddress(
+      network,
+      3,
+      3,
+      Accounts.WALLET_1.stxAddress
+    );
+    expect(poxAddrInfo0).toBeNull();
+
+    await expectAccountToBe(
+      network,
+      Accounts.WALLET_1.stxAddress,
+      100_000_000_000_000 - fee * 2,
+      0
+    );
+  });
+
+  it("stacking by other user should activate pox for cycle #3 (in cycle #2)", async () => {
+    const network = new StacksTestnet({ url: orchestrator.getStacksNodeUrl() });
+    let chainInfo = await getChainInfo(network);
+    blockHeight = chainInfo.burn_block_height;
+    const cycles = 1;
+
+    // Cloe stacks 80m
+    let response = await broadcastStackSTX(
+      {
+        poxVersion: 3,
+        network,
+        account: Accounts.WALLET_3,
+        fee,
+        nonce: chloeNonce++,
+      },
+      { amount: 80_000_000_000_000, blockHeight, cycles }
+    );
+    expect(response.error).toBeUndefined();
+    await asyncExpectStacksTransactionSuccess(orchestrator, response.txid);
+
+    let poxInfo = await getPoxInfo(network);
+    expect(poxInfo.current_cycle.id).toBe(2);
+    expect(poxInfo.next_cycle.stacked_ustx).toBe(80_000_000_000_000);
+  });
+
+  it("re-check locked amounts (cycle #3)", async () => {
+    const network = new StacksTestnet({ url: orchestrator.getStacksNodeUrl() });
+
+    // Wait for Stacks genesis block
+    await orchestrator.waitForNextStacksBlock();
+
+    // move on to the next cycle after unlock (N+5)
+    await waitForNextRewardPhase(network, orchestrator, 5);
+
+    let poxInfo = await getPoxInfo(network);
+    console.log(poxInfo);
+    // Assert that the current cycle has 80m STX locked
+    // and no STX locked for next cycle
+    expect(poxInfo.current_cycle.id).toBe(3);
+    expect(poxInfo.current_cycle.stacked_ustx).toBe(80_000_000_000_000);
+    expect(poxInfo.current_cycle.min_threshold_ustx).toBe(29_290_000_000_000);
+    expect(poxInfo.next_cycle.stacked_ustx).toBe(0);
+    expect(poxInfo.current_cycle.is_pox_active).toBe(true);
```

Comment:
This fails. Shouldn't this be true?

Reply:
It used to pass. But the pox activation is not clear to me. Are we in the correct poch?